proposals/2022-ambassadors.md

```diff
@@ -0,0 +1,55 @@
+# Proposal for budget and process for OWASP Ambassadors 
+
+## What is the problem we are trying to solve
+
+It seems that there is limited understanding of OWASP around the wider developer community. Whilst people may have heard of the top 10, it is not clear that they are familar with other projects which could help them or that they can/should be more involved in OWASP events.
+
+## Aim
+
+To increase awareness and understanding of OWASP among the wider developer community.
+
+In particular:
+* Encourage new members to join by highlighting the benefits
+* Encourage new corporate supporters by highlighting the importance of the organization
+* Encourage greater involvement in the local chapter
+* Showcase the variety of projects and resources which OWASP provides
+
+## Proposal Summary
+
+Creation of an OWASP ambassador role who would attend meetups and conferences and provide more in-depth information about getting into OWASP. 
+
+## Proposal Details
+
+To qualify as an OWASP ambassador, the person should commit to going to four non-OWASP events per year with the aim of advocating for OWASP and track this for audit on renewal. This would be a purely volunteer position and new applicants would have to be able to demonstrate previous OWASP involvement as well as having OWASP membership.
+
+In return, the ambassador would receive:
+* Specific recognition and thanks
+* Supplies of very basic/inexpensive OWASP swag such as stickers or pens to give out
+* Some dedicated OWASP apparel to make them stand out.
+* Coordination with the local chapter
+
+### OWASP stall
+
+That should also be a budget for a certain number of larger events per year where OWASP could have a table or stall for attendees to visit and learn more. 
+
+An OWASP ambassador would have to specifically request this  provide the following:
+* Sufficient early warning to prepare and plan
+* Justification for OWASP's presence at the event.
+* A plan for who would staff the table (could be coordinated with the local chapter)
+* A quote for having the additional SWAG below prepared locally rather than having to ship it from elsewhere.
+
+They could then be provided with extra resources (in addition to those noted above) including:
+* OWASP tablecloth
+* Larger information flyers about OWASP
+* Some sort of additional swag (like a unique t-shirt) to be dedicated for people who become OWASP members on the spot
+
+
+## Budget required
```

Comment:
I would think this is where per year, the budget is set by the Board. 

- The money is set at the start of the year and it's up to the out reach committee to plan and prioritize to list and identify different people and approaches. 
- It is up to the committee to choose if a select ambassador will attend or if there is a trusted member of a local chapter near the conference 
- It is up to the committee to say if a conference should have a booth, if we are sending people and engaging with the conference to see if we can obtain booths at lower costs 
-

Reply:
Bullet 1 - I added this just below and included a point on metrics.
Bullet 2 - I dislike the idea of ambassadors incurring travel expenses to attend an event except potentially for the very largest events which in turn should only be the most experienced/highly regarded ambassadors.
Bullet 3 - Added some clarification below